PokemonGo.RocketAPI/Helpers/Utils.cs

```diff
@@ -9,5 +12,37 @@ public static ulong FloatAsUlong(double value)
             var bytes = BitConverter.GetBytes(value);
             return BitConverter.ToUInt64(bytes, 0);
         }
+
+        public static uint GenerateLocation1(byte[] authTicket, double lat, double lng, double alt)
+        {
+            var seed = BitConverter.ToUInt32(new xxHash(32, 0x1B845238).ComputeHash(authTicket), 0);
+            var xxh32 = new xxHash(32, seed);
+
+            var locationBytes = new List<byte>();
+            locationBytes.AddRange(BitConverter.GetBytes(lat).Reverse());
+            locationBytes.AddRange(BitConverter.GetBytes(lng).Reverse());
+            locationBytes.AddRange(BitConverter.GetBytes(alt).Reverse());
```

Comment:
Values are stored in little endian format in memory on most computer architectures, including x86, which is reverse order from a human perspective. For some reason Niantic decided to use big-endian order for their hashing function.